about/release-notes/index.md

```diff
@@ -17,6 +17,73 @@ GitHub and be notified by email whenever a new release is available. On the
 click `Watch`, select `Custom` and then check `Releases`.
 </Highlight>
 
+## TimescaleDB&nbsp;2.15.0 on 2024-05-08
+
+<Highlight type="note">
+This release contains performance improvements and bug fixes since
```

Comment:
```suggestion
This release contains performance improvements and bug fixes introduced since
TimescaleDB v2.14.2. Best practice is to upgrade at the next available opportunity.
```